resources/shellHelperFunctions.sh

```diff
@@ -178,11 +178,54 @@ set_npm_version () {
   npm version --allow-same-version --no-git-tag-version --no-commit-hooks "$VERSION_WITH_TAG"
 }
 
+NUM_RE='^[0-9]+$'
+
+RETRY_MAX=5
+
+# Configuration:  wait between 10 sec and 120 sec.
+
+# in seconds.
+RETRY_MAX_WAIT_RANGE=111
+# in seconds
+RETRY_MIN_WAIT=10
```

Comment:
These should be declared local to reattempt_if_failing, so they don't pollute the global variable space.